docs/config/README.md

```diff
@@ -148,15 +148,15 @@ min_time = 4
 ## Directory
 
 The `directory` module shows the path to your current directory, truncated to
-three parent folders. Your directory will also be truncated to the root of the
-git repo that you're currently in.
+three parent folders.
```

Comment:
@matchai What's our policy for the module description text in config docs? Do we want to describe the default behavior? The minimal behavior? Everything?

Reply:
I think it would make sense to document the default behavior. The functionality of config options could also be explained if they are complex enough not to be understood from a small snippet in the config table.

Reply:
In this case, I would probably leave the description as it initially was, as this new configuration could be understood with a short description.